### DISCUSSION TITLE
OOP2 not getting or sending data

### DISCUSSION THREAD

**Discussion by mmarko12** [NightscoutFoundation/xDrip#3324] [Q&A] [answered]
Hi guys i recentlle put new libre 2 sensor and try to connect on xdrip with him like i do in usual, but OOP2 is not sending any data and when I open him on Bg values i dont see any data. I tried installing new version and uninstalling and installing all again but nothing changed. So if someone now how to fix that problem I will be grateful. This  second sensor that i cant connect. Thanks in advance.

**Comment by Navid200** [comment]
@PhilTU

**Comment by PhilTU** [comment]
Where are you and what versions of OOP2 & xDrip do you have.

Is the official app / device out of the picture?

**Reply by mmarko12** [reply]
I'm in Croatia
Version of OOP2 is 8a0ee8f-2022.06.10
Version of xdrip 2db9aa6-2024.01.16
Official app working fine i had lost signal when i try to connect with oop and xdrip but i manage to return it. And yeh i tried some other oop2 version like one that have "use direct intect" that this one don't have.

**Reply by PhilTU** [reply]
You need later version 05.2023

https://drive.google.com/file/d/1f1VHW2I8w7Xe3kSQqdaY3kihPLs47ILS/view

**Reply by PhilTU** [reply]
You need to disable official app if having problems with xDrip+

Running two is not supported.

**Reply by mmarko12** [reply]
I installe that version of oop disabled app and nothing, xdrip function when i scan him with nfc he shows value but oop is dead he doesn't show any signs of activity.

**Reply by PhilTU** [reply, marked as answer]
OK so no Bluetooth signal. If you have BG reading from NFC then OOP2 is OK. 

Have a look at https://xdrip.readthedocs.io/en/latest/troubleshoot/libre2direct/

in particular the "Bluetooth pairing" setting shown on the Bluetooth Status screen should be "Disabled". Check also that the NFC settings in xDrip+ for the Libre 2 are set to "always start sensor".

Reviewing the error/event log can be helpful, after NFC scan you should see "Bluetooth streaming enabled".

**Reply by mmarko12** [reply]
Thank you, I did all of that long time ago and looked event log and now somehow i tried just one more time with manual scanning sensor and now I finali get connection with oop and i see in app bg value. 
Thank you one more time for spending your time on this.
